src/Apps/PriceDatabase/__tests__/PriceDatabaseApp.jest.tsx

```diff
@@ -27,9 +29,11 @@ describe("PriceDatabaseApp", () => {
     })
 
     wrapper = mount(
-      <HeadProvider>
-        <PriceDatabase />
-      </HeadProvider>
+      <SystemContextProvider relayEnvironment={createMockEnvironment() as any}>
```

Comment:
This test was failing with:
```
 RUNS  src/Apps/PriceDatabase/__tests__/PriceDatabaseApp.jest.tsx
/Users/ozzievasdi/code/force/node_modules/relay-runtime/lib/query/fetchQueryInternal.js:268
  requestCachesByEnvironment.set(environment, requestCache);
                             ^

TypeError: Invalid value used as weak map key
```

After debugging I discovered that the `relayEnvironment` was not mocked out and failed this test. Im not clear on why this was working in v14 and doesn't in v18, perhaps due to a more strict check in more recent versions? In any case, providing context with `relayEnvironment` made this test pass.

Reply:
This works fine, but can also use `<MockBoot relayEnvironment={...}>` as that's more idiomatic (and will remove the need for `any`)

Reply:
Done in https://github.com/artsy/force/pull/12243/commits/eeb9658d7a9e5cd0d3fe755cf803d16d86000943